### ISSUE TITLE
A11Y: Table block accessiblity

### ISSUE THREAD

**Issue by tisto** [plone/volto#1504]
https://www.w3.org/WAI/tutorials/tables/

# ToDo

* [x] Write a Cypress test that creates a simple table block and checks if the result is accessible (https://github.com/plone/volto/blob/master/cypress/tests/core/a11y.js)
* [x] Fix issues that came up. One by one, referencing this ticket.
* [x] Question: do we end up with an accessible table when the editor marks a table cell (in the middle of the table) as "header cell"

![Add-Document (1)](https://user-images.githubusercontent.com/632105/117117403-cb705f80-ad8f-11eb-8368-50ee26112ab9.jpg)

**Comment by ThomasKindermann** [comment]
i added a simple cypress test that creates a table and tests for a11y - it results in 8 violations.
thats 2 more than on the frontpage. I will test a11y by hand to see what kind of violations it shows.

https://github.com/kitconcept/fzj-internet/compare/table-a11y-test?expand=1

**Comment by ThomasKindermann** [comment]
a11y test for table block: https://github.com/plone/volto/pull/2486

**Comment by ThomasKindermann** [comment]
#1504: https://github.com/plone/volto/pull/2568/
